index.js

```diff
@@ -1,10 +1,17 @@
 var _ = require('lodash');
 var loaderUtils = require('loader-utils');
 
+function getOptions(context) {
+  if (context.options && context.options.ejsLoader) {
+    return context.options.ejsLoader;
+  }
+  return {};
+}
+
 module.exports = function(source) {
   this.cacheable && this.cacheable();
   var query = loaderUtils.parseQuery(this.query);
-  var options = this.options ? this.options.ejsLoader || {} : {};
+  var options = getOptions(this);
```

Comment:
What about using `getOptions` from [loader-utils](https://github.com/webpack/loader-utils#getoptions).

It seems to be the standard way of getting the options from the loader.
That would require updating the `loader-utils` dependency (0.2.7 to 1.1.0) and remove the need for `var query = loaderUtils.parseQuery(this.query);` as well.

Reply:
We can do that in a second PR. I would love to have this out of the window fast.

Reply:
Yes what I suggest is definitely a bigger change